packages/components/src/Offer/Form/Bid.tsx

```diff
@@ -297,19 +303,30 @@ const OfferFormBid: FC<Props> = (props) => {
           <InputGroup>
             <NumberInput
```

Comment:
If I put a point, it trigger a BigNumber error in the useMemo of totalPrice.
Need to check all other inputs for similar issue as well.